Formula/flif.rb

```diff
@@ -0,0 +1,57 @@
+class Flif < Formula
+  desc "Formatted C++11/14/17 stdlib man pages from cppreference.com"
+  homepage "http://flif.info/"
+  head "https://github.com/FLIF-hub/FLIF.git"
+
+  stable do
+    patch :DATA
+    url "https://github.com/FLIF-hub/FLIF/archive/v0.2.2.tar.gz"
+    sha256 "92c893271a96550a31b1cc447db1739fee655c4b0c53f3b93f3f0d4f5292ec0c"
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "libpng"
+  depends_on "sdl2"
+
+  def install
+    make_args = %W[
+      PREFIX=#{prefix}
+    ]
+    cd "src" do
+      system "make", *make_args
+      system "make", "install", *make_args
+    end
```

Comment:
Or even `system "make", "install", "PREFIX=#{prefix}"` since the top-level `Makefile` uses the one in `src` anyway: https://github.com/FLIF-hub/FLIF/blob/fe2c1558a7245d9efaafd082f6237e9e06b72e2d/Makefile#L7-L8